app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -186,6 +191,7 @@ function DocumentCapture({ isAsyncForm = false, onStepChange = () => {} }: Docum
             onStepChange={setStepName}
             onStepSubmit={trackSubmitEvent}
             autoFocus={!!submissionError}
+            titleFormat="%{step} - Login.gov"
```

Comment:
is there a way we can get this derived from `APP_NAME` constant on the server?

Reply:
So that information is provided to the frontend in a couple of places, namely in the `<head>` and `<meta>` tags. 

I could easily grab it from the `<meta>` field like so:
`const appName = document.querySelector('meta[name="description"]')?.getAttribute('content');`

And do that only in the `document_capture.tsx` file. 

I suppose we could also add it as a new field to the translation .yml files? But that would be multiple changes where here we can just grab it from the HTML. 

Is this a reasonable suggestion @zachmargolis?

Reply:
I just went to pull up the spot where we pass attributes to the client and confirmed we are already passing the APP_NAME constant: https://github.com/18F/identity-idp/blob/7622467ec0482e4443431ca1d3ab0f19aa81a59d/app/views/idv/shared/_document_capture.html.erb#L14

so we could add it to [AppRootData](https://github.com/18F/identity-idp/blob/7622467ec0482e4443431ca1d3ab0f19aa81a59d/app/javascript/packs/document-capture.tsx#L24)

or alternatively, it looks like we pull this in a few places via `getConfigValue('appName')`

https://github.com/18F/identity-idp/blob/7622467ec0482e4443431ca1d3ab0f19aa81a59d/app/javascript/packages/document-capture/components/hybrid-doc-capture-warning.tsx#L28

Reply:
Yeah I'd suggest `getConfigValue('appName')` here.

Reply:
Excellent. Thanks for pointing me to that function @zachmargolis @aduth